docs/tutorials/mastering.md

```diff
@@ -1,60 +1,646 @@
 # Mastering
 
-In this tutorial, we will deploy and run the SD-Core 5G core network following Control and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also be deployed on an isolated cluster.
-
-```{image} ../images/sdcore_networking.png
-:alt: Network diagram
-:align: center
-```
-
-To complete this tutorial, you will need four machines (virtual or physical), with access to the networks as follows.
-
-| Machine | CPUs | RAM | Disk | Networks |
-| ------- | ---- | --- | ---- | -------- |
-| Control Plane Kubernetes Cluster     | 4 | 8g | 40g | `management` |
-| User Plane Kubernetes Cluster        | 4 | 4g | 20g | `management`, `access`, `core` |
-| Juju Controller + Kubernetes Cluster | 2 | 8g | 40g | `management` |
-| gNB Simulator Kubernetes  Cluster    | 4 | 4g | 20g | `management`, `ran` |
+In this tutorial, we will deploy and run the SD-Core 5G core network following Control and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also be deployed on an isolated cluster. [Multipass](https://multipass.run/) is used to create separate VMs connected with [LXD](https://ubuntu.com/lxd) networking.
 
+## 1. Prepare Host
 
-## Sample Values
-
-For the purpose of this guide, the following values will be used.
+A machine running Ubuntu 22.04 with the following resources:
+  - At least one NIC with internet access
+  - 8 cores
+  - 24 GB RAM
+  - 150 GiB disk  
 
 ### Networks
 
+The following networks will be used to create test environment.
+
 | Name | Subnet | Gateway IP |
 | ---- | ------ | ---------- |
 | `management` | 10.201.0.0/24 | 10.201.0.1 |
 | `access`     | 10.202.0.0/24 | 10.202.0.1 |
 | `core`       | 10.203.0.0/24 | 10.203.0.1 |
 | `ran`        | 10.204.0.0/24 | 10.204.0.1 |
 
+
+Create local network bridges to be used by LXD by adding below configuration under `/etc/netplan/50-sdcore-networks.yaml`. `enp4s0` is the name of the interface used in this tutorial, but you should replace it and the DHCP settings according to your environment.
+
+```console
+cat << EOF | sudo tee /etc/netplan/50-sdcore-networks.yaml
+# /etc/netplan/50-sdcore-networks.yaml
+network:
+  ethernets:
+    enp4s0: # Set the correct interface name
```

Comment:
I am getting so confused by GitHub.  I swear I already stated this somewhere, but cannot find my comment anymore.

As this is on the host, and we are creating a new netplan .yaml file, we should not repeat the user's ethernet configuration here.  Lines 31-34 can be removed

Reply:
50-sdcore-networks.yaml is a new file that we are creating. 
As enp4s0 is referenced in mgmt-br, we can not remove interface name. 
```
    mgmt-br:
      addresses:
        - 10.201.0.14/24
      routes:
        - to: default
          via: 10.201.0.1
          metric: 110
      interfaces:
        - enp4s0
```
Otherwise we will get following error:
```
/etc/netplan/50-sdcore-networks.yaml:3:3: Error in network definition: unknown key 'network'
  network:
```

Reply:
I am removing ethernets: section together with interfaces  for mgmt-bridge which seems useless.

Reply:
Done